### DISCUSSION TITLE
Welcome to Showcasing-2024-25-SPS Discussions!

### DISCUSSION THREAD

**Discussion by Reaper-1709** [Reaper-1709/Showcasing-2024-25-SPS#1] [Announcements]
## **Welcome!**
  _We’re using Discussions as a place to connect with other members of our school. We hope that you:_
  * Ask questions you are wondering about.
  * Share ideas regarding this project.

**Comment by dharmik2319** [comment]
i didnt even know this existed lol

**Comment by dharmik2319** [comment]
WE ARE FINALLY DONE
